### PR TITLE
[stable/pomerium] Bump pomerium image to 0.5.2

### PR DESCRIPTION
**Pull request by travisgroth**
#### Is this a new chart

No

#### What this PR does / why we need it:

Updates default Pomerium version to 0.5.2

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/pomerium/values.yaml [hunk 1]
```diff
@@ -134,7 +134,7 @@ imagePullSecrets: ""
 
 image:
   repository: "pomerium/pomerium"
-  tag: "v0.5.1"
+  tag: "v0.5.2"
```
Comment (davidkarlsen) [inline stable/pomerium/values.yaml:137]: Please make the readme match too: https://github.com/helm/charts/blob/master/stable/pomerium/README.md

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Review by desimone — APPROVED** [review]
LGTM other than what @davidkarlsen already pointed out 👍

**Review by desimone — APPROVED** [review]
/lgtm

**Pull request merged**